README/ReleaseNotes/v634/index.md

```diff
@@ -2,9 +2,14 @@
 % 2024-11
 <a name="TopOfPage"></a>
 
+## Important note about this development release
+
+6.34 is a short term support cycle not meant to be used for data taking. It will be superseded by the 6.36 cycle, which is foreseen to start with 6.36.00 in the second quarter of 2025. Patch releases of the 6.36 cycle will be provided until June 30th 2025.
```

Comment:
I believe the last sentence talks about the 6.34 cycle; we expect patch releases much longer for the data taking release 6.36, right?